iwf/unregistered_client.py

```diff
@@ -204,7 +209,7 @@ def start_workflow(
         workflow_start_response = handler_error_and_return(
             post_api_v1_workflow_start.sync_detailed(
                 client=self.api_client,
-                json_body=request,
+                body=request,
```

Comment:
All API functions no longer accept `json_body`, it has been renamed to just `body`